src/router/index.js

```diff
@@ -98,6 +99,13 @@ const routes = [
     component: ProductsDetails,
     meta: { title: "Add product", requiresAuth: true },
   },
+  {
+    path: "/products/:productId",
+    name: "Product Details",
+    props: true,
+    component: ProductsDetailsById,
```

Comment:
How about renaming this to `ProductDetailsRead`

Reply:
For me its not clear what Read is, i would go with the current naming its more conventional